cmd/restic/cmd_prune.go

```diff
@@ -362,10 +364,24 @@ func prune(opts PruneOptions, gopts GlobalOptions, repo restic.Repository, usedB
 		return err
 	}
 
+	// packs that are referenced in the index but are not needed can be ignored
```

Comment:
I think it would be helpful to add a remark to the comment that at this point `indexPack` only contains pack files missing from the repository.